lib/index.js

```diff
@@ -30,8 +30,16 @@ var Iterable = module.exports = integration('Iterable')
  */
 
 Iterable.prototype.track = function(payload, fn){
+  var endpoint;
+  if (payload["isUpdateCart"]) {
```

Comment:
Also, the mapper doesn't seem to be automatically used; I found I had to explicitly call the mapper.